vars/common.yml

```diff
@@ -0,0 +1,12 @@
+hushline_user: hushlineuser
```

Comment:
We have a var `hushline_user` set here but we hardcode the string `hushlineuser` in a bunch of places.

1. We should check that vars are reused and not hardcoded
2. The user should just be `hushline` since the `user` part is implied